kitsune/tags/south_migrations/0001_add_buddyup_tags.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        categories = [
+            'email', 'clock', 'camera', 'audio', 'performance', 'radios', 'sms', 'calls',
+            'browser', 'storage', 'geolocation', 'keyboard', 'system-updates', 'language',
+            'marketplace', 'other'
+        ]
+
+        for category in categories:
+            name = 'category:{}'.format(category)
+            orm['taggit.tag'].objects.create(name=name, slug=name)
```

Comment:
so this creates the slug with the colon in it. i don't think that should cause any issues, but it's different than how the slug is when it's created by the API.

API creates:
```python
tag.name = 'category:clock'
tag.slug = 'categoryclock'
```